astacus/coordinator/plugins/clickhouse/steps.py

```diff
@@ -435,6 +466,27 @@ async def run_on_every_node(
     await asyncio.gather(*[gather_limited(per_node_concurrency_limit, fn(client)) for client in clients])
 
 
+async def wait_for_condition_on_every_node(
```

Comment:
That's a good thing to add :+1: 

Can you write a specific test for the function itself ?